### DISCUSSION TITLE
Should I create a batch file to simplify installation?

### DISCUSSION THREAD

**Discussion by RowanHarley** [OpenBB-finance/OpenBB#401] [Ideas]
I could try to create a simple batch file to do the whole installation so that all people have to do is open the batch file and let it do it's thing. I don't think it would take much work, and would probably make it that bit more accessible.

**Comment by Jaramah** [comment]
I would really appreciate it as I had downloaded the zip file but i have no idea on how to carry on from there

**Comment by Tuff-Madman** [comment]
would be awesome for a lot of apes
